e2e/parallel-reflection-resolver/src/NamespacedSomeClassFound.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace E2e\Parallel\Reflection\Resolver;
```

Comment:
The namespace need to be in composer.json dev-autoload to verify that namespaced class exists.